Makefile

```diff
@@ -1,43 +1,26 @@
-CONFIGUREFLAGS=--enable-tests
+.PHONY: release build install uninstall clean test doc reindent
 
-# OASIS_START
-# DO NOT EDIT (digest: a3c674b4239234cbbe53afe090018954)
+release:
+	jbuilder build @install
 
-SETUP = ocaml setup.ml
+build:
+	jbuilder build @install --dev
 
-build: setup.data
-	$(SETUP) -build $(BUILDFLAGS)
+install:
+	cp ./_build/default/src/squeezed.exe ./squeezed.native
```

Comment:
I think it would be much simpler if we left the default `jbuilder install`, `jbuilder uninstall` recipes and modified the spec file to consume ` _build/install/default/bin/squeezed`. I find it easy to lose track with all the renaming.

Reply:
You are right, we should move all the install information here and cleanup the spec file as we did with most of the other ports. Well spotted

Reply:
@mseri , I am a little confused here. Tina is asking to keep Makefile here simple, but have spec file do the renaming and copying thing, in that the internal build structure is also exposed to spec file; your comments seem to agree with Tina, however you are saying to 'move all the install information here and cleanup the spec file', which seems to be conflict with what Tina asks. What's your actual suggestion here?

Reply:
What I meant is that you don't need to do any renaming. Leave it as is with its public name `squeezed` and modify the spec file to copy `_build/install/default/bin/squeezed`. There is no reason why the spec file should need a file called "squeezed.native".

Reply:
Ah, I've just looked at the spec file and I see what Marcello is saying (because we need to copy the executable to a different location). I agreed, these lines form the spec file can move here. Please have a look at the Makefiles of `xenopsd` or `message-switch` for examples on how this is done.